lib/ansible/modules/cloud/amazon/iam_role.py

```diff
@@ -355,7 +363,8 @@ def main():
             path=dict(default="/", type='str'),
             assume_role_policy_document=dict(type='json'),
             managed_policy=dict(type='list', aliases=['managed_policies']),
-            state=dict(choices=['present', 'absent'], required=True)
+            state=dict(choices=['present', 'absent'], required=True),
+            purge_policies=dict(type='bool', default=True),
```

Comment:
Should we be defaulting to True here? I know it's current behaviour but I'd think we might want to protect people from unintentional self-harm:

https://twitter.com/stigglor/status/888221882360582145